src/deepsparse/server/README.md

```diff
@@ -37,11 +37,30 @@ Usage: deepsparse.server [OPTIONS] COMMAND [ARGS]...
     - task: question_answering
       route: /unpruned/predict
       model: zoo:some/zoo/stub
+      name: question_answering_pipeline_1
     - task: question_answering
       route: /pruned/predict
       model: /path/to/local/model
+      name: question_answering_pipeline_2
   \```
-
+  
+  Optionally, to manually specify the set of loggers, define a 
```

Comment:
The `--help` text printed for `deepsparse.server` should be updated to include this additional text or it should be included outside of the `--help` text in this document